### PR TITLE
SEOImages: User Story 1463880, Part 28

### PR DESCRIPTION
**Pull request by damabe**
## Summary

Updating image file names and references to improve SEO.

Fixes #1463880)

Contributes to #10839

### CODE REVIEW

**Comment by damabe** [comment]
@mairaw This PR is ready for review. CATS report: https://contentqacats.azurewebsites.net/CheckResults/2019032000027/Report/

**Comment by damabe** [comment]
Updated based on other PR reviews. CATS: https://contentqacats.azurewebsites.net/CheckResults/2019032600058/Report/

**Comment by damabe** [comment]
Updated an altText with a property name. CATS: https://contentqacats.azurewebsites.net/CheckResults/2019032800047/Report/

File: docs/framework/winforms/advanced/how-to-rotate-reflect-and-skew-images.md [hunk 1]
```diff
@@ -24,15 +24,15 @@ You can rotate, reflect, and skew an image by specifying destination points for
   
  The following illustration shows the original image and the image mapped to the parallelogram. The original image has been skewed, reflected, rotated, and translated. The x-axis along the top edge of the original image is mapped to the line that runs through (200, 20) and (110, 100). The y-axis along the left edge of the original image is mapped to the line that runs through (200, 20) and (250, 30).  
   
- ![Stripes](./media/stripes1.gif "Stripes1")  
+ ![Skewed, reflected, translated, and rotated illustration.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-illustration.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-rotate-reflect-and-skew-images.md]:
```suggestion
 ![The original image and the image mapped to the parallelogram.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-illustration.gif)  
```

File: docs/framework/winforms/advanced/how-to-rotate-reflect-and-skew-images.md [hunk 2]
```diff
@@ -24,15 +24,15 @@ You can rotate, reflect, and skew an image by specifying destination points for
...
  The following illustration shows the original image and the image mapped to the parallelogram. The original image has been skewed, reflected, rotated, and translated. The x-axis along the top edge of the original image is mapped to the line that runs through (200, 20) and (110, 100). The y-axis along the left edge of the original image is mapped to the line that runs through (200, 20) and (250, 30).  
   
- ![Stripes](./media/stripes1.gif "Stripes1")  
+ ![Skewed, reflected, translated, and rotated illustration.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-illustration.gif)  
   
- The following illustration shows a similar transformation applied to a photographic image.  
+ The following illustration shows a similar transformation applied to a photographic image:  
   
- ![Transformed Climber](./media/transformedclimber.png "TransformedClimber")  
+ ![Skewed, reflected, translated, and rotated photo.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-photo.png)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-rotate-reflect-and-skew-images.md]:
```suggestion
 ![The picture of a climber and the picture mapped to the parallelogram.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-photo.png)  
```

File: docs/framework/winforms/advanced/how-to-rotate-reflect-and-skew-images.md [hunk 3]
```diff
@@ -24,15 +24,15 @@ You can rotate, reflect, and skew an image by specifying destination points for
...
+ The following illustration shows a similar transformation applied to a photographic image:  
   
- ![Transformed Climber](./media/transformedclimber.png "TransformedClimber")  
+ ![Skewed, reflected, translated, and rotated photo.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-photo.png)  
   
- The following illustration shows a similar transformation applied to a metafile.  
+ The following illustration shows a similar transformation applied to a metafile:  
   
- ![Transformed Metafile](./media/transformedmetafile.png "TransformedMetafile")  
+ ![Skewed, reflected, translated, and rotated metafile.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-metafile.png)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-rotate-reflect-and-skew-images.md]:
```suggestion
 ![Illustration of shapes and text and that mapped to the parallelogram.](./media/how-to-rotate-reflect-and-skew-images/reflected-skewed-rotated-metafile.png)  
```

File: docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md [hunk 4]
```diff
@@ -22,7 +22,7 @@ When you create a <xref:System.Drawing.Pen>, you can supply the pen width as one
   
 -   Set the value of the <xref:System.Drawing.Pen.Alignment%2A> property to <xref:System.Drawing.Drawing2D.PenAlignment.Center> (the default) to specify that pixels drawn with the green pen will be centered on the theoretical line. The following illustration shows the resulting line.  
   
-     ![Pens](./media/pens1a.gif "pens1A")  
+     ![Illustration showing green pixels centered on line.](./media/how-to-set-pen-width-and-alignment/green-pixels-centered-line.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md]:
```suggestion
     ![A black thin line with green highlight.](./media/how-to-set-pen-width-and-alignment/green-pixels-centered-line.gif)  
```

File: docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md [hunk 5]
```diff
@@ -35,7 +35,7 @@ When you create a <xref:System.Drawing.Pen>, you can supply the pen width as one
   
      The following illustration shows the resulting rectangle.
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md]:
```suggestion
     The following illustration shows the resulting rectangle:
```

File: docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md [hunk 6]
```diff
@@ -35,7 +35,7 @@ When you create a <xref:System.Drawing.Pen>, you can supply the pen width as one
   
      The following illustration shows the resulting rectangle.  
   
-     ![Pens](./media/pens2.gif "pens2")  
+     ![Illustration showing green pixels centered on a rectangle.](./media/how-to-set-pen-width-and-alignment/green-pixels-centered-rectangle.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md]:
```suggestion
     ![A rectangle drawn with black thin lines with green highlight.](./media/how-to-set-pen-width-and-alignment/green-pixels-centered-rectangle.gif)  
```

File: docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md [hunk 7]
```diff
@@ -49,7 +49,7 @@ When you create a <xref:System.Drawing.Pen>, you can supply the pen width as one
   
      Now the pixels in the wide green line appear on the inside of the rectangle as shown in the following illustration.  
   
-     ![Pens](./media/pens3.gif "pens3")  
+     ![Illustration showing green pixels inside a rectangle.](./media/how-to-set-pen-width-and-alignment/green-pixels-inside-rectangle.gif)
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md]:
```suggestion
     ![A rectangle drawn with black lines with the wide green line inside.](./media/how-to-set-pen-width-and-alignment/green-pixels-inside-rectangle.gif)  
```

File: docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md [hunk 8]
```diff
@@ -49,7 +49,7 @@ When you create a <xref:System.Drawing.Pen>, you can supply the pen width as one
   
      Now the pixels in the wide green line appear on the inside of the rectangle as shown in the following illustration.
```
Comment (mairaw) [inline docs/framework/winforms/advanced/how-to-set-pen-width-and-alignment.md]:
```suggestion
     Now the pixels in the wide green line appear on the inside of the rectangle as shown in the following illustration:
```

File: docs/framework/wpf/controls/bulletdecorator.md [hunk 9]
```diff
@@ -13,7 +13,7 @@ ms.assetid: 1756cabf-59b4-47a7-883d-1bdf2e5abe00
   
  The following illustration shows examples of controls that use a <xref:System.Windows.Controls.Primitives.BulletDecorator>.  
   
- ![3 BulletDecorators: CheckBox, RadioButton, TextBox](./media/bulletdecorator.png "BulletDecorator")  
+ ![Screenshot showing three bullet decorators.](./media/bulletdecorator/three-bullet-decorators.png)
```
Comment (mairaw) [inline docs/framework/wpf/controls/bulletdecorator.md]:
```suggestion
 ![Example of a Checkbox, a RadioButton, and a TextBox bullet decorator.](./media/bulletdecorator/three-bullet-decorators.png)  
```

File: docs/framework/wpf/controls/how-to-position-a-tooltip.md [hunk 10]
```diff
@@ -28,16 +28,19 @@ This example shows how to specify the position of a tooltip on the screen.
...
+  
+ ![Diagram showing ToolTip placement by using the Placement property.](./media/how-to-position-a-tooltip/tooltip-placement-property.png)
+ 
+ The following image shows tooltip placement by using the Placement and PlacementRectangle properties:   
+
+ ![Diagram showing ToolTip placement by using a PlacementRectangle property.](./media/how-to-position-a-tooltip/tooltip-placement-rectangle-property.png)  
+ 
+ The following image shows tooltip placement by using the Placement, PlacementRectangle, and Offset properties:   
+  
+ ![ToolTip placement by using the Offset property.](./media/how-to-position-a-tooltip/tooltip-placement-offset-property.png)
```
Comment (mairaw) [inline docs/framework/wpf/controls/how-to-position-a-tooltip.md]:
```suggestion
 ![Diagram showing ToolTip placement by using the Offset property.](./media/how-to-position-a-tooltip/tooltip-placement-offset-property.png)
```

**Review by mairaw — CHANGES_REQUESTED** [review]
Looks good @damabe. A few suggestions.

**Comment by damabe** [comment]
@mairaw Updates have been made. New CATS: https://contentqacats.azurewebsites.net/CheckResults/2019040200050/Report/

**Review by mairaw — APPROVED** [review]
Thanks!

**Pull request merged**